Makefile

```
@@ -74,8 +76,8 @@ build: build-ui ## Build bifrost-http binary
 run: build ## Build and run bifrost-http (no hot reload)
```

checkmake (0.2.2)

[warning] Target body for "run" exceeds allowed length of 5 (7).

(maxbodylength)